superset/migrations/versions/49b5a32daba5_add_report_schedules.py

```diff
@@ -119,13 +120,17 @@ def upgrade():
     )
 
 
+def has_unique_constraint(constraint_name: str, table_name: str) -> bool:
+    bind = op.get_bind()
+    inspector = Inspector.from_engine(bind)
+    unique_constraints = inspector.get_unique_constraints(table_name)
+    return constraint_name in {constraint["name"] for constraint in unique_constraints}
```

Comment:
I wonder if we should move this to `superset/utils/` and use it in all our new migrations?

Reply:
is constraint guaranteed to have a key of "name" for every engine?

Reply:
I just looked it up, that looks good.

Reply:
@betodealmeida it seems like we have similar logic like this [elsewhere](https://github.com/apache/superset/blob/62a8f2e193da3bdf1f43be085b5814be3dd476fa/superset/utils/core.py#L675-L684).